rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskController.java

```diff
@@ -202,6 +203,90 @@ public ResponseEntity<TaskRepresentationModel> claimTask(
+   *     the Task is in
+   * @title Request a review on a Task
+   */
+  @PostMapping(path = RestEndpoints.URL_TASKS_ID_REQUEST_REVIEW)
+  @Transactional(rollbackFor = Exception.class)
+  public ResponseEntity<TaskRepresentationModel> requestReview(@PathVariable String taskId)
+      throws InvalidTaskStateException, TaskNotFoundException, InvalidOwnerException,
+          NotAuthorizedException {
+    Task task = taskService.requestReview(taskId);
+    return ResponseEntity.ok(taskRepresentationModelAssembler.toModel(task));
+  }
+
+  /**
+   * This endpoint force request a review on the specified Task.
+   *
+   * @param taskId taskId the id of the relevant Task
+   * @return the Task after a review has been requested
+   * @throws InvalidTaskStateException if the state of the Task with taskId is not CLAIMED
+   * @throws TaskNotFoundException if the Task with taskId wasn't found
+   * @throws InvalidOwnerException if the Task is claimed by another user
+   * @throws NotAuthorizedException if the current user has no READ permissions for the Workbasket
+   *     the Task is in
+   * @title Force request a review on a Task
+   */
+  @PostMapping(path = RestEndpoints.URL_TASKS_ID_REQUEST_REVIEW_FORCE)
+  @Transactional(rollbackFor = Exception.class)
+  public ResponseEntity<TaskRepresentationModel> forceRequestReview(@PathVariable String taskId)
+      throws InvalidTaskStateException, TaskNotFoundException, InvalidOwnerException,
+          NotAuthorizedException {
+    Task task = taskService.forceRequestReview(taskId);
+    return ResponseEntity.ok(taskRepresentationModelAssembler.toModel(task));
+  }
+
+  /**
+   * This endpoint request changes on the specified Task.
+   *
+   * @param taskId the id of the relevant Task
+   * @return the Task after changes have been requested
+   * @throws InvalidTaskStateException if the state of the Task with taskId is not IN_REVIEW
+   * @throws TaskNotFoundException if the Task with taskId wasn't found
+   * @throws InvalidOwnerException if the Task is claimed by another user
+   * @throws NotAuthorizedException if the current user has no READ permissions for the Workbasket
+   *     the Task is in
+   * @title Request changes on a Task
+   */
+  @PostMapping(path = RestEndpoints.URL_TASKS_ID_REQUEST_CHANGES)
+  @Transactional(rollbackFor = Exception.class)
+  public ResponseEntity<TaskRepresentationModel> requestChanges(@PathVariable String taskId)
+      throws InvalidTaskStateException, TaskNotFoundException, InvalidOwnerException,
+          NotAuthorizedException {
+    Task task = taskService.requestChanges(taskId);
+    return ResponseEntity.ok(taskRepresentationModelAssembler.toModel(task));
+  }
+
+  /**
+   * This endpoint force requests changes on a Task.
+   *
+   * @param taskId the Id of the Task on which a review should be requested
+   * @return the change requested Task
+   * @throws InvalidTaskStateException if the state of the Task with taskId is not IN_REVIEW
```

Comment:
It doesn't hold because of the force option. How should we describe the Exception?